package.json

```diff
@@ -19,6 +19,7 @@
     "node-resque": "0.11.5",
     "pg": "4.1.1",
     "q": "1.1.2",
+    "redis": "0.12.1",
```

Comment:
Had to add this as a dependency for the __heartbeat__ endpoint health check. No big deal since node-resque was including it anyways.